### DISCUSSION TITLE
SGE syntax

### DISCUSSION THREAD

**Discussion by DustinSokolowski** [nextflow-io/nextflow#5282] [Q&A]
Hey!

I am trying to run TOGA on the SGE cluster, which relies on nextflow. My cluster has some relatively specific cluster options (see `clusterOptions`, but the amount of memory needs to be passed into nextflow `memory = "${_MEMORY_}G"`. I am unable to pass the memory command into clusterOptions (e.g.,  `clusterOptions = { "-V -l h_vmem "${_MEMORY_}G" -V -P simpsonlab -l h_stack=32M -l h_rt=48:00:00" }`. Do you know the syntax for this?

Thank you
Dustin

```
process {
    executor = "sge"
    penv = "smp"
    memory = "${_MEMORY_}G"
    cpus = '1'
    time = '24h'
    clusterOptions = { "-V -l h_vmem=64G -V -P simpsonlab -l h_stack=32M -l h_rt=48:00:00" }
}

executor {
    name = "sge"
    queueSize = 1000
    queueStatInterval = "10s"
}

```

**Comment by bentsherman** [comment]
Not sure if you're asking a Nextflow question or an SGE question. But here is how Nextflow defines the memory options:

https://github.com/nextflow-io/nextflow/blob/5a01f2779df64921406761e9cf4c92ea695b0d16/modules/nextflow/src/main/groovy/nextflow/executor/SgeExecutor.groovy#L78-L82
